plugin/bufsurf.vim

```diff
@@ -112,15 +113,52 @@ function s:BufSurfList()
     for l:bufnr in w:history
         let l:buffer_name = bufname(l:bufnr)
         if bufnr("%") == l:bufnr
-            let l:buffer_name = "* " . l:buffer_name
+            let l:buffer_name = "*>" . l:buffer_name
```

Comment:
Nitpicking here; I think it looks a bit better if there is a space behind the `>`. Would you object to that?

Reply:
No problem. Only reason I didn't was to keep the spacing the same as it was before. With the change there will be 3 chars before the buffer name instead of 2.